README.md

```diff
@@ -1,3 +1,50 @@
-# scrapy_capstone
-# scrapy_capstone
-# scrapy_capstone
+#Parse GridDynamics Blog with Scrapy and visualize it
+## Introduction
+   In this project a parser will be created for GridDynamics blog together with a report that will show us:
+      
+    - Top-5 Authors,
+    - Top-5 New Articles,
+    - Plot with articles counter of 7 most popular tags
+## Technologies
+    - Python 3.9
+    - Scrapy
+    - matplotlib
+    - pandas
+    - os
+    - yaml
+## Features
+The aim of this project is to create a Web Crawler that will parse https://blog.griddynamics.com/,
+obtain articles and author information from it.
+Then generate a report with this information:
+
+    - Top-5 Authors (based on articles counter)
+    - Top-5 New Articles (based on publish data)
+    - Plot with counts of 7 popular tags
+## Sources 
+    https://app.pluralsight.com/library/courses/scrapy-extracting-structured-data/table-of-contents
+    https://app.pluralsight.com/library/courses/data-visualization-with-python-introduction/table-of-contents
+    https://app.pluralsight.com/library/courses/python-fundamentals/table-of-contents
+
+<img width="633" alt="Screenshot" src="https://user-images.githubusercontent.com/75839583/103350362-025ffd00-4aa0-11eb-8cce-d58d50f8110b.png">
+
+## Environments setup and first run
+First create a virtual environment
+```
+python3 -m venv scrapy_capstyone
+```
+Before to run your code be sure that the following libraries are present, otherwise run the following commands:
+```
+pip3 install -r requirements.txt
+```
+Add `report` folder to your PATH in order to make all custom module imports valid using
+```
+cd scrapy_capstone && export PATH=$PATH:$PWD/report && echo $PATH
+```
+
+Finally run the following instruction from the command line in order to execute the report generator
+```
+python3 report.py
```

Comment:
Check if you committed all needed files, because I see this:
```
Traceback (most recent call last):
  File "/Users/dorlyakov/src_dev/scrapy_capstone/.venv/bin/scrapy", line 10, in <module>
    sys.exit(execute())
  File "/Users/dorlyakov/src_dev/scrapy_capstone/.venv/lib/python3.7/site-packages/scrapy/cmdline.py", line 114, in execute
    settings = get_project_settings()
  File "/Users/dorlyakov/src_dev/scrapy_capstone/.venv/lib/python3.7/site-packages/scrapy/utils/project.py", line 69, in get_project_settings
    settings.setmodule(settings_module_path, priority='project')
  File "/Users/dorlyakov/src_dev/scrapy_capstone/.venv/lib/python3.7/site-packages/scrapy/settings/__init__.py", line 287, in setmodule
    module = import_module(module)
  File "/usr/local/Cellar/python@3.7/3.7.9_2/Frameworks/Python.framework/Versions/3.7/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 962, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'report.settings'; 'report' is not a package
```
Check if you committed all __init__.py files